west.yml

```diff
@@ -124,7 +124,8 @@ manifest:
       groups:
         - hal
     - name: hal_stm32
-      revision: 51b373cd3455b8c2b9babbf6ff41918116a442ac
+      remote: carbon
+      revision: carbon/master-v2
```

Comment:
can we pin to a specific revision rather than a branch?

Reply:
Sure. I changed it to a commit hash. I suppose I should put the change on `master` so it's easier to find though...